src/dex-ui/store/walletSlice/walletSlice.ts

```diff
@@ -42,6 +42,16 @@ const createWalletSlice: WalletSlice = (set, get): WalletStore => {
         tokenData?.find((token: TokenBalanceJson) => token.tokenId === tokenId)?.decimals ?? defaultDecimals;
       return BigNumber(tokenAmount).shiftedBy(decimals).integerValue();
     },
+    doesUserHaveGODTokensToVote: () => {
+      const { wallet } = get();
+      const doesUserHaveGodTokens =
+        Number(wallet.pairedAccountBalance?.tokens.find((token) => token.tokenId === GovernanceTokenId)?.balance) > 1;
```

Comment:
A user can have a fraction of a governance token. This would be a rare case but it's technically possible. The condition should be `gov token balance> 0`.

Reply:
That's correct, I had a discussion with team and it was concluded that user needs atleast of 1GOD token to vote.
 
Right now the discussion is pending whether it is 1% or 1 quantity.